packages/e2e-tests/specs/editor/various/block-switcher.test.js

```diff
@@ -1,130 +0,0 @@
-/**
- * WordPress dependencies
- */
-import {
-	hasBlockSwitcher,
-	getAvailableBlockTransforms,
-	createNewPost,
-	insertBlock,
-	pressKeyWithModifier,
-} from '@wordpress/e2e-test-utils';
-
-describe( 'Block Switcher', () => {
-	beforeEach( async () => {
-		await createNewPost();
-	} );
-
-	it( 'Should show the expected block transforms on the list block when the blocks are removed', async () => {
```

Comment:
How about making "Should" a lower case along with other tests?

Reply:
I think it's okay. We don't really have a strict guideline when it comes to naming.